keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -952,33 +952,36 @@ when the feature is disabled.
 
 ### Rollout, Upgrade and Rollback Planning
```

Comment:
Could we do this as a separate PR? If someone is taking 3063 in this release, this may be a bit challanging to coordinate the rebase.

Reply:
I doubt that anyone besides me is going to touch 3063 in 1.32.